doc/platform_api/cmis_management_state_machine.md

```diff
@@ -0,0 +1,138 @@
+# CMIS management state machine HLD
+
+# High Level Design Document
+## Rev 0.1
+
+# Table of Contents
+- [1. Revision](#1-revision)
+- [2. Scope](#2-scope)
+- [3. Motivation](#3-motivation)
+- [4. Abbreviations](#4-abbreviations)
+- [5. Introduction](#5-introduction)
+- [6. HLD design](#6-hld-design)
+  - [6.1 The Requirement](#61-the-requirement)
+  - [6.2 CMIS Management Platform Support](#62-cmis-management-platform-support)
+  - [6.3 Modules Management Thread](#63-modules-management-thread)
+  - [6.4 General Module Detection Flow](#64-general-module-detection-flow)
+  - [6.5 Module Detection Flow States And Description](#65-module-detection-flow-states-and-description)
+  - [6.6 Module Detection Flow States And Functions](#66-module-detection-flow-states-and-functions)
+
+# 1. Revision
+| Rev | Date | Author | Change description |
+|:----------:|:----------:|:--------------:|:----------------------:|
+| 0.1 | 23/10/2023 | Doron Barashi | Initial version|
+| | | | |
+
+# 2. Scope
+This document provides high level design for CMIS management state machine flow

```

Comment:
not sure i understand the scope.
is there an hld of cmis state machine already? if not, how come the feature was introdcued?
i would like that rather of creating new one to update an exisitng.
if there is nothing we should only provide Prince what we did which is vendor specific

i would cont reading providing comments but i do want to understand why we need to do the below